test/test-prefix.js

```diff
@@ -0,0 +1,50 @@
+path = require('path');
+
+function checkPrefix(data, category, errors, prefix, path="") {
+  for (var key in data) {
+    if (key === "prefix" && typeof(data[key]) === "string") {
+      if (data[key].includes(prefix)) {
+        var error = `${prefix} prefix is wrong for key: ${path}`;
+        var rules = [
+          category == "api" && data[key].startsWith(prefix),
```

Comment:
For API, the prefix can be mixed case:
<dl>
<dt>Mozilla Firefox</dt>
<dd><code>moz</code>, <code>Moz</code></dd>
<dt>Chrome and Safari</dt>
<dd><code>webkit</code>, <code>WebKit</code> (and maybe even <code>Webkit</code> or <code>webKit</code>)</dd>
<dt>IE/Edge</dt>
<dd><code>ms</code>, <code>MS</code></dd>
</dl>

Reply:
FWIW, `Webkit` and `webKit` do not currently appear in Chrome's code base.